t/job_metrics.t

```diff
@@ -6,14 +6,15 @@ use FindBin;
 use lib "$FindBin::Bin/lib";
 use Test::Spec;
 use Test::Exception;
+use Time::HiRes qw(usleep);
 
 use HTFeed::JobMetrics;
 
 describe "HTFeed::JobMetrics" => sub {
     # SETUP start
     my $jm = HTFeed::JobMetrics->get_instance;
     my $item_metric = "ingest_pack_items";
-    my $byte_metric = "ingest_pack_bytes";
+    my $byte_metric = "ingest_pack_bytes_write";
     my $invalid_metric = "not a valid metric";
     before each => sub {
 	$jm->clear() if defined $jm;
```

Comment:
I think this line is the first example of some tabs pre-existing/remaining in the .t file. I'd like to see an additional TIDY on this to clear out the rest of the tabs.

Reply:
Re-indented: https://github.com/hathitrust/feed/pull/128/commits/79b0b9577fb2217d8ade59a20a46a680c001b86d?w=0